barretenberg/cpp/src/barretenberg/benchmark/protogalaxy_bench/protogalaxy.bench.cpp

```diff
@@ -30,7 +30,7 @@ void fold_one(State& state) noexcept
     std::shared_ptr<Instance> instance_1 = construct_instance();
     std::shared_ptr<Instance> instance_2 = construct_instance();
 
-    auto folding_prover = composer.create_folding_prover({ instance_1, instance_2 }, composer.commitment_key);
```

Comment:
it was a method on composer and i didn't need to pass ck which is its own field as argument